org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2.metainfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2</id>
```

Comment:
LV2 shouldn't be part of the ID.

Reply:
There's a non-LV2 version of this plugin over at https://github.com/sdatkinson/NeuralAmpModelerPlugin. It's also from a different maintainer. My intention was to disambiguate the LV2 version from the non-LV2 version.

Given the above, should I still drop the LV2 bit? :thinking:

Reply:
no. the idea would be to add the format to the same package if anything.